src/kendo/Bar.js

```diff
@@ -0,0 +1,95 @@
+"use strict";
+(function (root, factory) {
+    if (typeof define === "function" && define.amd) {
+        define(["jquery","kendo.all.min","../common/HTMLWidget"], factory);
+    } else {
+        root.kendo_Bar = factory();
+    }
+}(this, function (jquery, kendo, HTMLWidget) {
+
+
+
+    function Bar() {
+        HTMLWidget.call(this);
+
+        var context = this;
+        this._tag = "div";
+        this._type = "unknown";
+        this._chartType = "BarChart";
+    }
+    Bar.prototype = Object.create(HTMLWidget.prototype);
+    Bar.prototype.constructor = Bar;
+    Bar.prototype._class += " kendo_Bar";
+
+
+    Bar.prototype.init = function (domNode, element) {  };
+
+
+    Bar.prototype.update = function (domNode, element) {
+
+        var data = this.data();
+        var kendoCategories = [];
+        var kendoSeries = [];
+        this.data().forEach(function(ele, ind, arr){
+            kendoCategories.push(ele[0]);
+
+        });
+
+        this.columns().forEach(function(ele, ind, arr){
+            if(ind > 0){
+
+                var correspondingData = [];
+                data.forEach(function(dEle, dInd, dArr){
+                   correspondingData.push(dEle[ind]);
+                });
+                kendoSeries.push({name:arr[ind], data:correspondingData});
+            }
+        });
+
+        var create = function() {
+
+
+                var kendoChartOptions = {
+                    legend: {
+                        visible: false
+                    },
+                    seriesDefaults: {
+                        type: "bar"
+                    },
+                    series:kendoSeries,
+
+                    valueAxis: {
+                        max: 100,
+                        line: {
+                            visible: false
+                        },
+                        minorGridLines: {
+                            visible: true
+                        },
+                        labels: {
+                            rotation: "auto"
+                        }
+                    },
+                    categoryAxis: {
+                        categories: kendoCategories,
+                        majorGridLines: {
+                            visible: false
+                        }
+                    },
+                    tooltip: {
+                        visible: true,
+                        template: "#= series.name #: #= value #"
+                    }
+                };
+
+                $("#"+domNode.id).kendoChart(kendoChartOptions);
+
+        };
+
+        $(document).ready(create);
+        $(document).bind("kendo:skinChange", create);
```

Comment:
The creation code for kendo should go into ```Bar.prototype.enter``` which has the same signature as update.

In general when writing a new widget, there are three methods of interest:
* enter
* update
* exit

Enter:  This is called once when the widget "enters" the web page - so is a good place to create and append your charts.
Exit:  This is called once when the widget "exits" the web page - so a good place to clean up any pieces before the hosting dome node is destroyed.
Update:  This is typically called when the "data",  "published property" or "size" is changed.  So typically its where you update the data within the kendo chart and update any visual settings that you may have "published" to the DSP user.